src/Containers/FileUpload/FileUpload.tsx

```diff
@@ -0,0 +1,267 @@
+import React,{useCallback,useState,useRef,useEffect} from 'react'
+import styled from 'styled-components'
+import {flex} from '@Utils/Mixins'
+import {TextLayout} from '@Layouts'
+import {useDropzone} from 'react-dropzone'
+import {Image} from '@styled-icons/fa-solid/Image';
+import {CheckCircle} from '@styled-icons/fa-solid/CheckCircle'
+import {TimesCircle} from '@styled-icons/fa-solid/TimesCircle'
+import {MainTheme} from '@Themes'
+import {Loading} from '../Loading/Loading'
+
+export interface IFileUploadProps{
+    title:string;
+    subTitle:string;
+    minHeight:number;
+    setBase64:(base64StringFile:string)=>void;
+    isUploading:boolean;
+    isSuccess:boolean;
+    isFailure:boolean;
+    successMessage:string;
+    failureMessage:string;
+}
+
+export const FileUpload:React.FC<IFileUploadProps>=({
+    title,
+    subTitle,
+    minHeight,
+    setBase64,
+    isUploading,
+    isSuccess,
+    isFailure,
+    successMessage,
+    failureMessage,
+}):React.ReactElement=>{
+    const [height,setHeight]=useState<number|undefined>(undefined)
+    const [totalHeight,setTotalHeight]=useState(0)
+    const [totalHeightPlus,setTotalHeightPlus]=useState(0)
+    const [padding]=useState(10)
```

Comment:
Why is this here? If your using this for conditional animation it should be in styled components. If your using this for rendering an animations you should use CSS Keyframes

Reply:
I use this padding state variable to calculate (compute) the width of the element and set it when its position is absolute.

Reply:
I think this is incorrect, if you can explain more about why it's being used / how that will help suggest a better solution. Are you sure you don't need a useRef or anything else I'm sure there is a better way